frontends/concrete-python/concrete/fhe/compilation/module_compiler.py

```diff
@@ -158,15 +160,6 @@ def evaluate(
                 artifact object to store informations in
         """
 
-        if self._is_direct:
```

Comment:
Isn't direct circuits supported now? If that's the case we should raise an exception instead of ignoring it.

Reply:
It is not supported for modules no. And documented nowhere if I am right !

Reply:
It's here:
https://docs.zama.ai/concrete/tutorials/direct_circuits

If it's hard to support, we can leave it out for now.